app.json

```diff
@@ -0,0 +1,49 @@
+{
+    "name": "BoxyHQ Enterprise SaaS Starter Kit",
+    "description": "The Open Source Next.js Enterprise SaaS Starter Kit. Next.js based SaaS starter kit that saves you months of development by starting you off with all the features that are the same in every product, so you can focus on what makes your app unique.",
+    "repository": "https://github.com/boxyhq/saas-starter-kit",
+    "logo": "https://boxyhq.com/img/logo.png",
+    "keywords": [
+        "saas",
+        "starter",
+        "kit",
+        "enterprise"
+    ],
+    "env": {
+        "NEXTAUTH_URL": {
+            "description": "Next.js authentication URL."
+        },
+        "NEXTAUTH_SECRET": {
+            "description": "Next.js authentication secret."
+        },
+        "SMTP_HOST": {
+            "description": "SMTP server host name."
+        },
+        "SMTP_PORT": {
+            "description": "SMTP server port number."
+        },
+        "SMTP_USER": {
+            "description": "SMTP server username."
+        },
+        "SMTP_PASSWORD": {
+            "description": "SMTP server password."
+        },
+        "SMTP_FROM": {
+            "description": "SMTP server sender's email address."
+        },
+        "DATABASE_URL": {
+            "description": "Hosted Database URL."
+        },
+        "APP_URL": {
+            "description": "Public root URL of the Enterprise SaaS installation, replace <HEROKU_APP_NAME> with the app name.",
+            "value": "https://<HEROKU_APP_NAME>.herokuapp.com"
+        },
```

Comment:
@Saptarshee94 Let's make the SVIX_URL, and SVIX_API_KEY as **optional**. These are required only if the app use the Webhook feature. The developer can add these value later from the Heroku env page.

Reply:
Resolved issue by making `SVIX_URL` and `SVIX_API_KEY `  optional.